### DISCUSSION TITLE
Allow selecting small ranges on Range Slider

### DISCUSSION THREAD

**Discussion by RodrigoGarciaLab** [mantinedev/mantine#4598] [Feature requests]
Right now is not possible to select a range shorter than 10 units. Is there any reason for this? If I create a Range Slider component with `min=0` and `max=20` it should be possible to select the range between 14 and 17 for example.

 
[screen-capture.webm](https://github.com/mantinedev/mantine/assets/31510776/e2cc3947-60e4-43d1-aad5-515be4ac9059)

**Comment by rtivital** [comment]
Set `minRange={1}`

**Reply by RodrigoGarciaLab** [reply]
@rtivital Thanks for the quick answer! Didn't know about that prop, it might be worth adding it to the documentation tab.
